packages/extras/package.json

```diff
@@ -69,6 +69,7 @@
     "types": "./dist/index.d.ts"
   },
   "dependencies": {
+    "three-mesh-bvh": "^0.7.1",
```

Comment:
I was wondering about this new dependency but yes, it is part of [drei's component](https://github.com/pmndrs/drei/blob/b03ed79a7aa7584f79fe689022d747d4272749fe/src/materials/MeshRefractionMaterial.tsx#L6). My guess is that's what it takes to make this kind of effect?

Reply:
Ah, probably should have mentioned that in the PR, yeah that what makes the raycasting in the shader work fast enough for real time performance, could work without it, but then it would probably be _very_ slow

Reply:
yeah, I'm a fan of `three-mesh-bvh` 👍 

wanted to point it out for other reviewers

Reply:
Oh, also on reflection, it might be an idea for us to talk about the dependencies of extras components. It would be more work though and we'd have to check/update all our other components too

Reply:
I'd prefer to depart from how drei does things here and instead opt for peer dependencies. This option would definitely require more documentation and education for setup but would also ultimately result in a better UX by reducing bundle size.

`three-mesh-bvh` is [~70kb](https://bundlephobia.com/package/three-mesh-bvh@0.7.1). It's a pretty commonly used library so if someone is using it and `MeshRefractionMaterial` with this dependency setup they're adding 140kb to their app, which is pretty costly.

If we use peer dependencies then the user would have to be instructed in the docs that installing `three-mesh-bvh` is required, but once they do so then they're only adding ~70kb of cost.

I understand that drei likely does things the way they do for convenience, but I think of the alternative as putting the end user first. Curious if anyone has any thoughts here.

Reply:
haha helping to guide towards a svelte'ier way of doing it 😄

Reply:
I'm fine with both options, but if going the peer dependency route, the documentation should mention that in multiple places in big bold warning letters so people don't miss it imo.

Reply:
^ Absolutely, I'd imagine we'd have to create a new component in the docs that very boldly spells out a dependency requirement

Reply:
I'm on @michealparks side here. Let's make it a peer dependency and add a `<Tip type="info">` as the first thing in the docs.

Reply:
Nice! sorted. I think we'll be all good with this going forward because even if folks miss the info at the top, the node error should direct them to why/what to do as well.